python/llm/example/CPU/PyTorch-Models/Model/Qwen-VL-Chat/README.md

```diff
@@ -0,0 +1,74 @@
+# Qwen-VL-Chat
+In this directory, you will find examples on how you could use BigDL-LLM `optimize_model` API to accelerate Qwen-VL-Chat models. For illustration purposes, we utilize the [Qwen/Qwen-VL-Chat](https://huggingface.co/Qwen/Qwen-VL-Chat) as a reference Qwen-VL-Chat model.
+
+## Requirements
+To run these examples with BigDL-LLM, we have some recommended requirements for your machine, please refer to [here](../README.md#recommended-requirements) for more information.
+
+## Example: Multimodal chat using `chat()` API
+In the example [generate.py](./generate.py), we show a basic use case for a Qwen-VL-Chat model to start a multimodal chat using `chat()` API, with BigDL-LLM 'optimize_model' API.
+### 1. Install
+We suggest using conda to manage the Python environment. For more information about conda installation, please refer to [here](https://docs.conda.io/en/latest/miniconda.html#).
+
+After installing conda, create a Python environment for BigDL-LLM:
+```bash
+conda create -n llm python=3.9 # recommend to use Python 3.9
+conda activate llm
+
+pip install --pre --upgrade bigdl-llm[all] # install the latest bigdl-llm nightly build with 'all' option
+pip install accelerate tiktoken einops scipy transformers_stream_generator==0.0.4 peft deepspeed torchvision matplotlib
```

Comment:
Here the additional packages should be based on https://github.com/QwenLM/Qwen-VL/blob/master/requirements.txt, which are not the same as you list here :)

Also it is recommended to add a line of comment here `# additional package required for Qwen-VL-Chat to conduct generation`